src/applications/toe/components/GetHelp.jsx

```diff
@@ -12,7 +12,7 @@ export default function GetHelp() {
         </a>{' '}
         If you have technical difficulties using this online application, call
         our MyVA411 main information line at{' '}
-        <va-telephone contact="8006982411" />. We're here 24/7.
+        <va-telephone contact="8006982411" extension="711" />. We're here 24/7.
```

Comment:
Hey, I don't think this is right. It will actually dial the extension as well. It's not just part of the view.